components/How_It_Works/HowItWorksElement/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+export interface HowItWorksElementProps {
+  title: string
+  summary: string
+  imgSrc: string
+  imgAlt: string
+  rowReverse?: true
+}
+
+const HowItWorksElement: React.FC<HowItWorksElementProps> = (props) => {
```

Comment:
Give a better name and filename/folder name without underscore

Reply:
How about WalloraWorking ?